tests/integration/defs/kv_cache/test_prefix_aware_scheduling.py

```
@@ -0,0 +1,749 @@
+        print_error(f"LMBenchmark exited with code {proc.returncode}")
+        for line in "".join(stderr_lines).strip().split("\n")[-10:]:
+            if line:
+                print_error(f"  [stderr] {line}")
+    return proc.returncode
+
+
+def _parse_csv_metrics(csv_path):
+    """Parse LMBenchmark output CSV and return TTFT metrics.
+
+    Returns a dict with keys:
+        num_requests  – total rows
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)